test/transform.js

```diff
@@ -23,7 +23,17 @@ test('transform adds sourcemap comment and uses cache on second time', function
 
     var es6ify = proxyquire('..', { './compile' : trackingCompile } )
 
-    var file = path.join(__dirname, '../example/src/features/iterators.js');
+    var sourceRoot = path.join(__dirname, '..', 'example', 'src');
+    var relPath = path.join('features', 'iterators.js');
+
+    var opts = {sourceRoot: sourceRoot};
```

Comment:
Thanks for pointing that out. Like I said I worked on those changes hastily, so I could push them before the weekend. I'll fix it.